Gemfile.lock

```diff
@@ -20,6 +20,7 @@ GEM
     faraday (0.9.2)
       multipart-post (>= 1.2, < 3)
     ffi (1.9.10)
+    ffi (1.9.10-x64-mingw32)
```

Comment:
these gemfile changes could probably be ignored. there are some version updates, but mainly I just struggled to get the thing running locally on my machine